### PR TITLE
Added some Resources

### PR DESCRIPTION
**Pull request by ronika-das**
=>Added several resources and corresponding links and divided them into:
1. Hackathons and Conferences
2. Open Source PRograms
3. Mentorship Programs
4. Coding Education

=>Pls let me know if I am heading in the right direction.
=>Some of these are global, but I have added under India Heading as of now

### CODE REVIEW

File: women-in-tech-resources/resources-list.md [hunk 1]
```diff
@@ -58,6 +58,30 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 
 ### Mailing Lists, Google or Facebook Groups, and Slacks:
 - [LinuxChix India](http://india.linuxchix.org/)
+- [Facebook Developer Circle - Various States](https://developers.facebook.com/developercircles)
 
 ### Meetups and Events for Women & Non-binary Engineers:
 - [Women Who Code - Delhi](https://www.womenwhocode.com/delhi)
+- [Google Developers Group - Various States](https://developers.google.com/programs/community/)
```
Comment (alodahl) [inline women-in-tech-resources/resources-list.md]: Are these groups free?

Comment (ronika-das) [reply women-in-tech-resources/resources-list.md]: yes. one can apply to join these groups as volunteers. They organize meetups, talks etc. One can know about current developments in tech and these are good for networking as well.

**Comment by ronika-das** [comment]
Thanks. I shall continue to find and add more

File: women-in-tech-resources/resources-list.md (hunk 1 above)
Comment (manaswinidas) [reply women-in-tech-resources/resources-list.md]: As far as I know, Women Techmakers too fits in this category. Just a suggestion.

Comment (ronika-das) [reply women-in-tech-resources/resources-list.md]: noted. i will add it in a new pull request among others

Comment (ronika-das) [reply women-in-tech-resources/resources-list.md]: i have added the WTM India page link . I will add more resources in a new pull request

**Comment by ronika-das** [comment]
> There are a plethora of opportunities that Women Techmakers provides. Please go through the link and see whether it fits in.

**Comment by manaswinidas** [comment]
Don't forget to squash your commits

**Comment by ronika-das** [comment]
how does one squash the commits?

**Comment by ronika-das** [comment]
hello mentors. i am unable to squash the commits through my cmd. pls let me know a solution to this. i have synchronised my repo with techtonica's repo. It's showing 29 commits now. kindly help

**Comment by ronika-das** [comment]
okay i think i have rebased. pls let me know if it worked :)

**Comment by alodahl** [comment]
@manaswinidas I can squash them once I merge. Once ronika is on the scoresheet, I will merge.

**Pull request merged**